dao/accumulo.rya/src/main/java/mvm/rya/accumulo/AccumuloRyaDAO.java

```diff
@@ -202,12 +202,18 @@ public void delete(Iterator<RyaStatement> statements, AccumuloRdfConfiguration c
                 //query first
                 CloseableIteration<RyaStatement, RyaDAOException> query = this.queryEngine.query(stmt, conf);
                 while (query.hasNext()) {
-                    deleteSingleRyaStatement(query.next());
+                	RyaStatement resolved = query.next();
```

Comment:
Nice fix.  Can you fix your whitespace in settings eclispe?  a tab made it in here.